webapp/ordering/cron.py

```diff
@@ -170,3 +170,22 @@ def do(self):
         if order_round.is_over and order_round.rides_mails_sent is False:
             print("Sending ride mails!")
             order_round.send_ride_mails()
+
+
+class SendPrepareRideMails(CronJobBase):
+    RUN_EVERY_MINS = 30
+
+    schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
+    code = 'ordering.send_prepare_ride_mails'
+
+    def do(self):
+        print("SendPrepareRideMails")
+        order_round = get_current_order_round()
+        print("Order round: %s" % order_round)
+        if order_round.is_open:
+            next_order_round = get_next_order_round()
+            print("Next order round: %s" % next_order_round)
+            if (next_order_round is not None and
```

Comment:
If the new order round is created after the current order round was closed, the mail will never be sent?

Reply:
Should it be `if order_round.is_over` on L185?